apps/web-app/pages/[orgSlug]/settings/user/profiles.vue

```diff
@@ -99,8 +99,9 @@
         '5xl'
       );
     }
-
     uploadLoading.value = false;
+    refreshNuxtData('getUserSingleProfileNav');
+    refreshNuxtData('getUserOrgsNav');
```

Comment:
why is this here?
we only need to refresh the user profile since thats what we are changing